project/Release.scala

```diff
@@ -29,7 +29,7 @@ object Release {
     val repo = extracted.get(Publish.defaultPublishTo)
     val state1 = extracted.runAggregated(publishSigned in projectRef, state)
     val (state2, Seq(api, japi)) = extracted.runTask(unidoc in Compile, state1)
-    val (state3, docs) = extracted.runTask(generate in Sphinx, state2)
+    val (state3, docs) = extracted.runTask(paradox in ProjectRef(projectRef.build, "akka-docs") in Compile, state2)
 
```

Comment:
It's generated in `akka-docs/target/paradox/site/main/`, I just want to make sure that we sync that directory and not `paradox/site/`